app/javascript/tests/mocks/mockPatients.js

```diff
@@ -278,8 +280,8 @@ const mockPatient1 = {
   user_defined_symptom_onset: null,
   was_in_health_care_facility_with_known_cases: false,
   was_in_health_care_facility_with_known_cases_facility_name: null,
-  white: null
-}
+  white: null,
```

Comment:
@holmesie can/should we standardize these eslint rules?
https://eslint.org/docs/rules/comma-dangle
https://eslint.org/docs/rules/semi

Reply:
Yeah, I'll create a ticket for it.